_posts/2021-02-09-january-rubygems-update.md

```diff
@@ -0,0 +1,37 @@
+---
+title: January 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in January.
+
+## RubyGems News
+
+In January, we released new versions of bundler `v2.2.7` and `v2.2.8`, and rubygems `v3.2.7` and `v3.2.8`, the most important change being to fix resolver edge cases and speed up bundler - [#4277](https://github.com/rubygems/rubygems/pull/4277) this change makes bundler faster and the resolver more correct.
+
+We also made the following improvements and fixes:
+
+- looked up information for a CI related issue where the the `--silent` flag did not work with `gem update --system` - [#2907](https://github.com/rubygems/rubygems/issues/2907#issuecomment-756957235) and that was resolved on [#3789](https://github.com/rubygems/rubygems/pull/3789)
+- worked on RFC for Bundler Version Locking -[#29](https://github.com/rubygems/rfcs/pull/29).
+- got pending releases out and kept working on more performance issues in bundler.
+
+This month, [RubyGems](https://github.com/rubygems/rubygems) gained [172 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2021-01-02%7D...master@%7B2021-01-31%7D), contributed by 12 authors. There were 53323 additions and 1646 deletions across 2565 files.
+
+## RubyGems.org News
+
+This month in RubyGems.org, we improved versions endpoint response time which makes SQL query on `compact_index` versions up to **1400x times** faster by adding index on `indexed` and `yanked_at` columns - [#2615](https://github.com/rubygems/rubygems.org/pull/2615).
```

Comment:
:thinking: AFAIK in the end the performance gain wasn't 1400x times better as per https://github.com/rubygems/rubygems.org/pull/2616#issue-560599453.

Reply:
yeah, please update it to 3x. 1400x was on my local setup and for the SQL query (not endpoint),  which doesn't have any load otherwise.

Reply:
Alright, thanks will update it.

Reply:
Perhaps we remove the response time thingy altogether. The real highlight of Jan was "reduced delay in update in versions endpoint after gem push from the worst-case of 3660 seconds to 60 seconds" https://github.com/rubygems/rubygems.org/issues/1877#issuecomment-772268568